src/offsetDistance.js

```diff
@@ -4,31 +4,22 @@
   var isNumeric = internalScope.isNumeric;
 
   function offsetDistanceParse (input) {
-    /* According to spec:
-       https://drafts.fxtf.org/motion-1/#offset-distance-property
-       initial value is 0 so will return 0 on undefined.
-       Will return a number when units is px and a number array if units is %.
-    */
-    if (input === undefined) {
-      return {value: 0, unit: 'px'};
-    }
-
     var distance = 0;
 
     if (input.endsWith('%')) {
       distance = input.substring(0, input.length - 1);
-      if (!(isNumeric(distance))) {
+      if (!(isNumeric(distance)) || distance.length === 0) {
```

Comment:
I would have put the length check on the left. Calling isNumeric is redundant if the length is 0.